.travis/debian10/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Can we re-use this script also for debian 7 and debian 9? Just move i to `.travis/run.sh`.

Reply:
@mliszcz , there are some diverses related to including coveralls and zmq_base. Apart from that we can think about re-use of course!

Reply:
Yes, but only debian 8 setup is different. For debian 7, 9 and 10 we have exactly the same file (except formatting, newlines, etc.)

Reply:
true

Reply:
we can pass target system as a variable into the script or, in case debian 8, use dedicated script file. I have no strong opinion on the approach

Reply:
I haven't noticed that we have a common call to the `run.sh` script:
```yaml
script:
  - .travis/${OS_TYPE}/run.sh
```

In this case, maybe we can just create soft links with relative paths like:
```bash
debian10/run.sh # actual file
debian9/run.sh -> ../debian10/run.sh
debian7/run.sh -> ../debian10/run.sh
```

Reply:
Well, that is unnecessary complication to my taste due to misleading unclear call chain. It even reads strange: `debian 7 -> debian 10` 

I would rather stay with variable passed into script or test `OS_TYPE` in travis and make a call accordingly, something like (pseudo code):

```
script:
  - IF ${OS_TYPE} == debian8 THEN ./debian8/run.sh ELSE ./run.sh

```

Reply:
Actually we can reuse all of the code and handle the differences not OS specfic but feature flag specific. I've pushed something which goes along that line.

@mliszcz Thanks for the commit squashing suggestions. I've used these and also squashed  one more.